src/cli/kube.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright © 2018 Atomist, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import { execFileSync } from "child_process";
+import * as stringify from "json-stringify-safe";
+
+import { webhookBaseUrl } from "../atomistWebhook";
+import {
+    Configuration,
+    getUserConfig,
+    resolveTeamIds,
+    resolveToken,
+} from "../configuration";
+
+function ghRawUrl(repo: string): string {
+    return `https://raw.githubusercontent.com/atomist/${repo}/master`;
+}
+
+/**
+ * Install the Atomist Kubernetes utilities in a Kubernetes cluster
+ * using kubectl.
+ *
+ * @param argv yargs processed command-line arguments
+ * @return 0 if successful, non-zero otherwise
+ */
+export async function cliAtomistKube(argv: any): Promise<number> {
+
+    const ns: string = argv.namespace;
+    const environment: string = (argv.environment) ? argv.environment : "kubernetes";
+
+    const userConfig = getUserConfig();
+    const token = resolveToken(userConfig);
+    if (!token) {
+        console.error(`No token set, try running 'atomist config' first`);
+        return Promise.resolve(1);
+    }
+    const teamIds = resolveTeamIds(userConfig);
+    if (!teamIds || teamIds.length < 1) {
+        console.error(`No Atomist workspace/team IDs set, try running 'atomist config' first`);
+        return Promise.resolve(1);
+    }
+
+    const k8hookBase = `${webhookBaseUrl()}/atomist/kube/teams`;
+    const webhooks = `${k8hookBase}/` + teamIds.join(`,${k8hookBase}/`);
```

Comment:
That's an interesting way for writing that. I think I would have done:

`teamsIds.map(id => ´${k8hookBase}/${id}´).join(",");`

Reply:
That would be better. I had this solution in my head because I used it for Helm, whose templating system does not have map.